apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -464,26 +464,23 @@ impl BridgeQueryPlanner {
                             UsageReportingComparisonResult::StatsReportKeyNotEqual
                                 | UsageReportingComparisonResult::BothNotEqual
                         ) {
-                            tracing::warn!(
-                                monotonic_counter.apollo.router.operations.telemetry.studio.signature = 1u64,
-                                generation.is_matched = false,
-                                "Mismatch between the Apollo usage reporting signature generated in router and router-bridge"
+                            u64_counter!(
+                                "apollo.router.operations.telemetry.studio.signature",
+                                "The match status of the Apollo reporting signature generated by the JS implementation vs the Rust implementation",
+                                1,
+                                "generation.is_matched" = "false"
```

Comment:
Shouldn’t this use a `false` boolean instead of a `"false"` string? Similarly for three other `is_matched` below.

Reply:
Hmm I'm sure I tried that before and it was causing a build error, but now it's not. I can update it in my next PR.